apps/web/src/client/components/html-content-editor/html-content-editor.js

```diff
@@ -43,13 +43,24 @@ function initHtmlContentEditor() {
 		}
 		const editBox = editor.getEditorElements().wwEditor;
 		const charCount = el.parentElement?.getElementsByClassName('character-count')[0];
+		const charCountWarning = el.parentElement?.getElementsByClassName('character-count-warning')[0];
 
 		function updateCharacterCount() {
 			if (!charCount) {
 				return;
 			}
 			const text = editBox.innerText || editBox.textContent || '';
 			charCount.textContent = text.length.toString();
+			if (!charCountWarning || !(charCountWarning instanceof HTMLElement)) {
+				return;
+			}
+			const limit = charCountWarning.dataset.characterCountLimit;
+			if (!limit) {
+				return;
+			}
+			const showWarning = text.length > parseInt(limit);
+			console.log('showWanring', { showWarning, length: text.length, limit: parseInt(limit) });
+			charCountWarning.style.setProperty('display', showWarning ? 'block' : 'none', 'important');
```

Comment:
have you considered using classes for this

Reply:
I did, and had a similar thing to do for the excerpt component, and decided on styles. I forget why!